SB2018/2018-04-12.md

```diff
@@ -1,13 +1,21 @@
 ---
 to: All Teams
-subject: Rules amendments
+subject: Rules amendments + Media Consent info
 ---
 
 Hi!
 
+We've got two more things of importance to mention:
+
+## Media Consent forms
+
+We'd like to clarify that Media Consent forms should be given to us on paper, not on a screen, as we need to file them.
```

Comment:
I'd prefer if it was stated that they should be brought to the competition:

> We'd like to clarify that Media Consent forms should be brought with you to the competition, on paper, rather than being emailed beforehand, or shown on a screen, as we need to file them.

Reply:
fixed in 
bf9bb6e